website/docs/quickstart_guide/docker.md

```diff
@@ -41,13 +43,17 @@ They are not suitable for most production use-cases because they keep all data i
    `ferretdb` runs FerretDB.
 
 2. Start services with `docker compose up -d`.
+   To be sure that you've fetched the latest version you can precede the command with `docker compose pull`.
 
 3. If you have `mongosh` installed, just run it to connect to FerretDB.
    If not, run the following command to run `mongosh` inside the temporary MongoDB container, attaching to the same Docker network:
 
    ```sh
-   docker run --rm -it --network=ferretdb --entrypoint=mongosh mongo mongodb://ferretdb/
+   docker run --rm -it --network=ferretdb --entrypoint=mongosh mongo mongodb://ferretdb/ -u username -p password --authenticationMechanism=PLAIN
    ```
 
-You can also install with FerretDB with the `.deb` and `.rpm` packages
+   FerretDB will use credentials passed in `mongosh` flags or MongoDB URI to authenticate to the PostgreSQL database.
+   See [Security#Authentication](/security.md#authentication) for more details.
```

Comment:
Did you check this link works? :hugs:

Reply:
I'm writing a bit more about it ATM, but let's use relative file paths as they seem to work everywhere, including editors/IDEs (Ctrl+click works) and Docusaurus. In that case, it should be `../security.md#authentication`.

Reply:
> Did you check this link works?

Yes

> let's use relative file paths as they seem to work everywhere, including editors/IDEs (Ctrl+click works) and Docusaurus

Good point, I'll change it then